org.adempiere.base/src/org/compiere/util/ResetCacheCallable.java

```diff
@@ -28,7 +28,17 @@ public class ResetCacheCallable implements Callable<Integer>, Serializable
 	private static final long serialVersionUID = 6669645804211785491L;
 	
 	private String tableName;
-	private int Record_ID;
+	private Integer Record_ID;
+	private String key;
+	
+	/**
+	 * @param tableName
+	 * @param key
+	 */
+	public ResetCacheCallable(String tableName, String key)
+	{
+		this.key = key;
```

Comment:
need here also the line:
```
this.tableName = tableName;
```

as there is no tablename is resetting the whole cache and that's why the unit test works